guacamole/src/main/webapp/app/client/types/ManagedClient.js

```diff
@@ -346,6 +346,18 @@ angular.module('client').factory('ManagedClient', ['$rootScope', '$injector',
                             ManagedClientState.ConnectionState.CONNECTING);
                         break;
 
+                    // Connection is being established
```

Comment:
Is being established?  Or is established?

Reply:
Oops. Definitely should be "is established".